src/jshint.js

```diff
@@ -4225,7 +4225,7 @@ var JSHINT = (function() {
     var i = 0;
     var inof = ["in", "of"];
     var level = 0; // BindingPattern "level" --- level 0 === no BindingPattern
-    var comma; // First comma punctuator at level 0
+    var firstcomma; // First comma punctuator at level 0
```

Comment:
How about we merge this in to fix the bug (or just my one rename to fix the issue with comma).  Then maybe someone in the future can do that refactoring.

All I want is jshint to work again and I can update without repatching the comma fix in every version.